.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Release
+
+on:
+  workflow_run:
+    workflows: ['CI']
+    types:
+      - completed
+  push:
+    branches:
+      # semantic-release valid branches
+      - '+([0-9])?(.{+([0-9]),x}).x'
+      - 'main'
+      - 'next'
+      - 'next-major'
+      - 'beta'
+      - 'alpha'
+
+jobs:
+  main:
+    name: NPM Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Use Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        uses: bahmutov/npm-install@v1
+        with:
+          useLockFile: false
+
+      - name: Build package
+        run: npm run build
```

Comment:
should we build in `ci.yml` ? I think we should (without releasing ofc)

Reply:
Mmmm not really sure about that, what would be the purpose for it? I'm gonna merge both workflows into a single file actually, since it's triggering the release twice.

Reply:
to ensure there are no compilation errors (Typings for instance) committed without noticing?

Reply:
Ah! I'm doing that on v4 so on "Code Validation" job I'll add format check and typescript check steps. I'll create a PR later with those updates for v4!